packages/flutter/test/material/text_field_test.dart

```diff
@@ -1721,7 +1721,41 @@ void main() {
     await skipPastScrollingAnimation(tester);
 
     scrollableState = tester.firstState(find.byType(Scrollable));
-    expect(scrollableState.position.pixels, isNot(equals(0.0)));
+    expect(scrollableState.position.pixels, equals(222.0));
```

Comment:
This tests horizontal scrolling.  The reason I changed this to an exact pixel value was to catch a mistake I made (adding the vertical `caretOffset` to the horizontal `caretRect.left`).

Reply:
It would be helpful to add a comment here, just to explain what the 222 means